tests/ui/conftest.py

```diff
@@ -95,29 +297,29 @@ def user(create_superuser, my_base_url, fxa_account, jwt_token):
 @pytest.fixture(scope='function')
 def live_server(request, transactional_db):
```

Comment:
Yeah, the fixture could generate a unique author, and if that information is returned then the test can use it for search/verification. We don't need to solve that here, but it's a way around having a live server for every test and may allow us to execute tests in parallel again. The only downside I see to this is that we wouldn't be able to assume that the data generated by the test fixtures is the only data in the application, but I don't see that being much of a restriction.